tests/ha/check_after_reboot.pm

```diff
@@ -10,7 +10,7 @@
 # Summary: Check cluster status *after* reboot
 # Maintainer: Loic Devulder <ldevulder@suse.com>
 
-use base 'opensusebasetest';
+use base 'sles4sap';
```

Comment:
I would prefer to still use `opensusebasetest` here and include `sles4sap` library.